apps/daffio/src/app/docs/components/table-of-contents/sidebar/sidebar-header/sidebar-header.component.html

```diff
@@ -1,3 +1,3 @@
-<daff-sidebar-header [dismissible]="true" (closeSidebar)="close()">
```

Comment:
this is a functional change, not a docs one. It should be in a separate PR

Reply:
Yeah this was a mistake. Removed.